File: OOPs/Constructors/parametarized_constructor.cpp
#include <bits/stdc++.h>
using namespace std;

class Car {
private:
    string brand;
    string model;
    int year;

public:
    Car(string b, string m, int y) {
        brand = b;
        model = m;
        year = y;
    }

    void displayInfo() {
        cout << "Car: " << brand << " " << model << ", Year: " << year << endl;
    }
};

int main() {
    Car myCar("Toyota", "Corolla", 2020);
    myCar.displayInfo(); 
    return 0;
}

File: OOPs/Constructors/ParametarizedConstructor.java
public class ParametarizedConstructor {
    String brand;
    String model;
    int year;

    public ParametarizedConstructor(String brand, String model, int year) {
        this.brand = brand;
        this.model = model;
        this.year = year;
    }

    void displayInfo() {
        System.out.println("Car: " + brand + " " + model + ", Year: " + year);
    }

    public static void main(String[] args) {
        ParametarizedConstructor myCar = new ParametarizedConstructor("Toyota", "Corolla", 2020);
        myCar.displayInfo(); 
    }
}
